css/properties/position-area.json

```diff
@@ -21,8 +21,7 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": false,
-              "impl_url": "https://bugzil.la/1838746"
+              "version_added": "131"
```

Comment:
```suggestion
              "version_added": "131",
              "partial_implementation": true,
              "notes": "While the CSS property is parsed and accepted, it does not have any effect."
```

Reply:
Technically it is a full implementation of `position-area`.
I shall add the reason to the note
> "The property is parsed and accepted, but it does not have any effect, as it also needs the implementation of `anchor-name` and `position-anchor`."